Lib/test/test_timeout.py

```diff
@@ -180,19 +179,15 @@ def testConnectTimeout(self):
         # timeframe).
 
         # For the records, the whitehole/blackhole configuration has been set
-        # up using the 'pf' firewall (available on BSDs), using the following:
+        # up using the 'iptables' firewall, using the following rules:
         #
-        #   ext_if="bge0"
+        # -A INPUT -p tcp --destination-port 56666 -j DROP
+        # -A INPUT -p udp --destination-port 56666 -j DROP
+        # -A INPUT -p tcp --destination-port 56667 -j RETURN
+        # -A INPUT -p udp --destination-port 56667 -j RETURN
```

Comment:
I don't understand well the RETURN rule. Is it an ACCEPT at the end?

> RETURN means stop traversing this chain and resume at the next rule in the previous (calling) chain. If the end of a built-in chain is reached or a rule in a built-in chain with target RETURN is matched, the target specified by the chain policy determines the fate of the packet.

I don't know the INPUT default policy.

Reply:
REJECT maybe an acceptable replacement here and is more explicit, I will check.

Reply:
Yep, that works :)